cmd/set/account_password/account_password.go

```diff
@@ -0,0 +1,91 @@
+package account_password
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+
+	"github.com/spf13/cobra"
+)
+
+var fileName string
+
+var inpUID string
+
+type Password struct {
+	inpPassword string `json:"password"`
+}
+type Result struct {
+	PasswordHash string `json:"PasswordHash"`
+}
+
+func Newaccount_passwordCmd() *cobra.Command {
+	opts := &Password{}
+	cmd := &cobra.Command{
+		Use:     "account_password",
+		Short:   "set account_password",
+		Long:    `This commmand sets account_password`,
+		Example: heredoc.Doc(`$ lr set account_password --inpUID <inpUID> --inpPassword <inpPassword>`),
```

Comment:
Seems like the example is wrong. please try it from your end once and then add it here.